en/components/grids_templates/summaries.md

```diff
@@ -324,6 +324,51 @@ class MySummary extends IgxNumberSummaryOperand {
 
 }
 
+### Summary Template
+`igxSummary` targets the column summary providing as a context the column summary results.
+
+```html
+<igx-column ... [hasSummary]="true">
+    <ng-template igxSummary let-summaryResult>
+        <span> My custom summary template</span>
+        <span>{{ summaryResult[0].label }} - {{ summaryResult[0].summaryResult }}</span>
+    </ng-template>
+</igx-column>
+```
+
+When a default summary is defined the height of the summary area is calculated by design depending on the column with the highest number of summaries and the display density of the grid. Overriding this height could be achieved through the exposed [summaryRowHeight]({environment:angularApiUrl}/classes/igxgridcomponent.html#summaryRowHeight) property. As an argument it takes a number and setting it to a falsy value would trigger the default summary row sizing behavior.
```

Comment:
Setting the rowSummaryHeight property to some falsy value will not hide the footer, it will trigger the default calculation for the summary area, so I will modify this part of your suggestion